hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/KeyPrefixRegionSplitPolicy.java

```diff
@@ -29,7 +29,11 @@
  *
  * This ensures that a region is not split "inside" a prefix of a row key.
  * I.e. rows can be co-located in a region by their prefix.
+ *
+ * @deprecated since 2.4.3 and will be removed in 4.0.0. Use {@link RegionSplitRestriction},
```

Comment:
You need to update this deprecation string to also include the applicable 2.3.x version number.